### DISCUSSION TITLE
Insert to note at cursor missing?

### DISCUSSION THREAD

**Discussion by MTGMAD** [logancyang/obsidian-copilot#820] [Q&A]
I noticed in the newest version of Copilot that the insert to note at cursor is missing, That has been one of my favorite features since it was introduced, is it just a mistake for it to be missing? or is that on purpose.  I already miss it.

**Comment by logancyang** [comment]
@MTGMAD it's still here just with a new icon 
<img width="200" alt="SCR-20241115-lluz" src="https://github.com/user-attachments/assets/95dd0a1d-00fb-4d1b-b60f-c2a44dcae6ba">

We need to be more consistent with our icons in the future 😬 cc @zeroliu

**Reply by zeroliu** [reply]
Sorry for the confusion. I couldn't find a matching icon when migrating to the new icon library so I picked an alternative that I personally find most relevant to the functionality. The icon migration is done now. I won't expect any changes to existing icons in the near term.

**Reply by MTGMAD** [reply]
Thank you! I think my panic took over, and I just looked for the icon!!
